RetailCoder.VBE/UI/Inspections/InspectionSeverityImageSourceConverter.cs

```diff
@@ -24,6 +24,11 @@ public class InspectionSeverityImageSourceConverter : IValueConverter
 
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
+            if (value == null)
+            {
+                throw new ArgumentException("Value parameter must be non-null");
```

Comment:
why bring the nukes out directly? Log an error and return a default. Don't blow it all to smithereens directly...

Reply:
`Possible 'System.NullReferenceException` was brought up. Thought this was the proper way of addressing that.